src/cmake/GeosxOptions.cmake

```diff
@@ -141,7 +141,7 @@ if( ENABLE_HYPRE AND ENABLE_HYPRE_CUDA )
     set( GEOSX_LOCALINDEX_TYPE "int" CACHE STRING "" )
     set( GEOSX_GLOBALINDEX_TYPE "long long int" CACHE STRING "" )
 else()
-    set( GEOSX_LOCALINDEX_TYPE "int" CACHE STRING "" )
+    set( GEOSX_LOCALINDEX_TYPE "std::ptrdiff_t" CACHE STRING "" )
```

Comment:
we want this to be `int`

Reply:
Yes i know i just wanted to check Jenkins was ok with that. Maybe i Can add a cast in the multiplication routine.

Reply:
Changed that back and used size_t when computing linear index, seems to be ok on pecan docker file.

Reply:
Ok I broke something else. This needs some work.